dpmmpython/dpmmwrapper.py

```diff
@@ -1,5 +1,5 @@
 import julia
-julia.Julia(compiled_modules=False)
```

Comment:
IIRC `compiled_modules=False` was added for either Python 2.7 support or a bug with PyCall when using Python 3.8.
The 2.7 support is no longer required, I will check if the problem still exists with PyCall and python 3.8 and merge if it is solved.

Reply:
See my view on this here https://github.com/JuliaPy/pyjulia/issues/425#issuecomment-788260938